README.adoc

```diff
@@ -61,6 +61,33 @@ If you want to connect devices on external networks to HiveMQ CE, please make su
 
 Just in time builds for current branches on this repository and for specific commits are available https://hivemq.github.io/nightly-builds/[here].
 
+== Run with Docker
+
+The current state of the `master` branch will be pushed to the https://hub.docker.com/r/hivemq/hivemq-ce[hivemq/hivemq-ce] repository.
```

Comment:
The current state of the `master` branch will be pushed to the
->
All releases as well as the current state of the `master` branch are available in the https://hub.docker.com/r/hivemq/hivemq-ce[hivemq/hivemq-ce] repository on dockerhub.